src/CommunityToolkit.Maui.Core/Interfaces/IAvatarView.shared.cs

```diff
@@ -0,0 +1,73 @@
+﻿namespace CommunityToolkit.Maui.Core;
+
+/// <summary>Avatar view interface.</summary>
+public interface IAvatarView : IBorderView, ILabel, Microsoft.Maui.IImage, IImageSource
+{
+	//// NOTE to implementer: Implement the following properties as public.
+
+	/// <summary>Gets a value indicating the avatar border colour.</summary>
+	Color BorderColor { get; }
+
+	/// <summary>Gets a value indicating the avatar border width.</summary>
+	double BorderWidth { get; }
+
+	/// <summary>Gets a value indicating the avatar corner radius (MAUI).</summary>
+	CornerRadius CornerRadius { get; }
+
+	//// NOTE to implementer: Implement these methods explicitly
```

Comment:
Do we need it?
At least right now it is in not a correct place

Reply:
Been removed.